integration_tests/tests/cucumber.rs

```diff
@@ -3828,18 +3766,18 @@ async fn receive_an_error(_world: &mut TariWorld, _error: String) {
 #[when(expr = "I have a lagging delayed node {word} connected to node {word} with \
                blocks_behind_before_considered_lagging {int}")]
 async fn lagging_delayed_node(world: &mut TariWorld, delayed_node: String, node: String, delay: u64) {
-    let mut config = BaseNodeConfig::default();
-    config.state_machine.blocks_behind_before_considered_lagging = delay;
+    let mut base_node_config = BaseNodeConfig::default();
+    base_node_config.state_machine.blocks_behind_before_considered_lagging = delay;
 
-    spawn_base_node_with_config(world, true, delayed_node, vec![node], config).await;
+    spawn_base_node_with_config(world, true, delayed_node, vec![node], base_node_config).await;
 }
 
 #[then(expr = "node {word} has reached initial sync")]
 async fn node_reached_sync(world: &mut TariWorld, node: String) {
     let mut client = world.get_node_client(&node).await.unwrap();
     let mut longest_chain = 0;
 
-    for _ in 0..(NUM_RETIRES * 2) {
+    for _ in 0..(TWO_MINUTES_WITH_HALF_SECOND_SLEEP * 11) {
```

Comment:
I'm just wondering, was this 22 minutes timeout in the `js` cucumber? Feels like it's way too much. Anyway we will probably (somewhere in the future) fine tunes these arbitrary values.

Reply:
Yup 22 minutes in the previous one too. 😓